spec/features/user_can_view_profile_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+require 'web_helpers.rb'
+
+feature "Profile page" do
+
+  scenario "user signs up and views profile" do
+    sign_up
+    # visit("/users/show")
```

Comment:
This test was started by the other pair in our team, who were working on displaying the profile. It will have been completed by them on their branch.